src/google/calendar.ts

```diff
@@ -0,0 +1,113 @@
+// import * as fs from 'fs';
```

Comment:
Should we delete this?

Reply:
Oh the entire file you mean, I was thinking we need the logic for sometime later? We can definitely write this again. Shouldn't be a big deal, since it's already saved in the commits somewhere.

Reply:
Deleted the calendar file